src/solidity-frontend/solidity_convert.cpp

```diff
@@ -199,6 +202,49 @@ bool solidity_convertert::convert()
   return false; // 'false' indicates successful completion.
 }
 
+void solidity_convertert::multi_json_file()
+{
+  // This function is used to handle multiple JSON files into a single JSON file
+  src_ast_json = src_ast_json_array[0];
```

Comment:
What does the `src_ast_json_array[0]` stand for? Could you please add some documentations here?

Reply:
Yes, src_ast_json_array[0] means the .sol file that is being verified and not being imported.